src/sass/base/_global.scss

```
@@ -0,0 +1,32 @@
+html{
+    width: fit-content;
+}
+
+body{
+    //font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+    line-height: 1;
+}
+
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+a{
```

```suggestion
a {
```
Check lại space giữa selector với dấu {